app/assets/stylesheets/application.scss

```
@@ -30,7 +31,29 @@ h2 {
   color: #D8D7D8;
 }
 
+.row {
+  align-items: stretch;
+  justify-content: space-evenly;
+}
+
 .card {
   background-color: #00567E;
   box-shadow: 10px 10px 5px 0px rgba(0,0,0,0.30);
+  margin: 10px;
+}
+
+.members {
+  display: flex;
+  }
+
+.member {
+  margin: 10px;
+  font-size: 14px;
+}
+
+.unstyled-list {
+  list-style: none;
+  margin: 5px;
```

```suggestion
  margin: 0;
```

This class says that it's unstyled, so let's make it actually unstyled.

---

> This class says that it's unstyled, so let's make it actually unstyled.

I removed the margin and the dashboard looks worse (the letters on names and disciplines from different team members are touching each other, making very difficult to distinguish and read). Not sure what's the best option at this point. Any ideas?